src/main/kotlin/it/polpetta/cli/Login.kt

```diff
@@ -35,21 +36,27 @@ import kotlin.system.exitProcess
 
 class Login @Inject constructor(private val jenkinsSession: JenkinsSession) :
     CliktCommand(help = "Login to the remote Jenkins instance") {
-    private val url: String by argument("server_url", "Jenkins server URL")
+    private val urlArgName: String = "server_url"
+    private val url: String? by argument(urlArgName, "Jenkins server URL").optional()
     private val username: String? by option("-u", "--username", help = "Jenkins username").prompt("Username")
     private val password: String? by option("-p", "--password", help = "Jenkins password/auth token").prompt("Password")
 
     override fun run() {
-        val jenkins = jenkinsSession.with(url, username, password)
+        var validUrl: String = url.orEmpty()
+        if (url == null || url.isNullOrBlank()) {
+            validUrl = "http://localhost/"
+            println("No $urlArgName provided, using default url $validUrl")
+        }
```

Comment:
This might be a little too compact but you could also do this:
```
val validUrl = url.takeIf { !it.isNullOrBlank() } ?: run {
            println("No $urlArgName provided, using default url")
            "http://localhost/"
        }
```
But yeah... I don't know how much is it worth it :)